test/fixtures/moby-runc.yml

```diff
@@ -67,7 +67,7 @@ sources:
 
 build:
   env:
-    CGO_ENABLED: 1
+    CGO_ENABLED: "1"
```

Comment:
We do need to be explicit now about string vs. int values for env variables in the yaml itself

Reply:
That's a major gotcha in YAML. I think it's worth investigating if there's a way you can get CUE to interpret values in certain structures as strings. I'm not sure that it's possible but it may be.